docs/content/docs/data-structures/overview.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Overview of Distributed Data Structures
+menuPosition: 1
+---
+
+The Fluid Framework provides developers with distributed data structures (DDSes) that automatically ensure that each
+connected client has access to the same state. The APIs provided by DDSes are designed to be familiar to programmers
+who’ve used common data structures before.
```

Comment:
dumb quotes please

Reply:
here and elsewhere

Reply:
What does this mean?

Reply:
```suggestion
who've used common data structures before.
```

Sorry, I wasn't clear. Does the code change clarify? I'll look into a lint rule for this.